Add draft content - "DDMoRe Language Community" and "...But I only use NONMEM"

MikeKSmith opened the pull request:
Added two new blog posts. 
DDMoRe Language Community - introduces the DDMoRe Language Community, its purpose, role and how it will interact with the DDMoRe Foundation. 
"...but I only use NONMEM" - to counter the argument that MDL isn't needed because most users only use NONMEM.

MikeKSmith-Pfizer committed:
Remove old .Rproj files. Add draft content.

Commit:
Update But_I_only_use_NONMEM.Rmd

Remove draft : "true" and insert Tibco as company that bought S-Plus from Insightful.

Commit:
Update DDMoRe_Language_Community.Rmd

Change "-" to ":".

Commit:
Update DDMoRe_Language_Community.Rmd

Change how contributors can contribute blog posts using the "draft" branch of the website repo.

Commit:
Update DDMoRe_Language_Community.Rmd

Spelling correction.

Commit:
Update But_I_only_use_NONMEM.Rmd

Minor edits

Commit:
Update But_I_only_use_NONMEM.Rmd

minor edit

vjd commented on content/post/But_I_only_use_NONMEM.Rmd:
```diff
@@ -0,0 +1,142 @@
+### I share via publications
+Writing down the maths of a model is a great way to share the model between
+quantitative disciplines. It's a universal language and brings with it rigour
+and a lack of ambiguity. ***BUT*** while two modellers can agree on the maths 
```
Maths comes with assumptions, and most authors do their best to state these assumptions. Can you imagine the horror when the implemented math is different from the assumptions that the author was making?

vjd commented on content/post/But_I_only_use_NONMEM.Rmd:
```diff
@@ -0,0 +1,142 @@
+## NONMEM doesn't do ***EVERYTHING***
+NONMEM and associated tools e.g. PsN have some limitations in available 
+functionality, for example optimal trial design. And while NONMEM ***can*** be
+used for prediction and simulation of new designs, regimens, scenarios there 
```
and scenarios,

vjd commented on content/post/But_I_only_use_NONMEM.Rmd:
```diff
@@ -0,0 +1,142 @@
+other technologies)](http://blog.revolutionanalytics.com/popularity/). So to 
+dismiss open-source solutions as "irrelevant" may be regarded as a 
+short-sighted view.
+
```
For a moment, just compare the growth our community in pharmacometrics made in comparison to other fields such as computational biology, genetics, ecology to name a few. The experts in these fields are scientists, similar to us, and not software engineers. Yet, they made massive strides in science mainly due to their adoption of open source methods and tools. Experts working in the industry put their developed tools and materials out in the open, a culture that pharmacometrics experts in the industry have yet to embrace after all these years. 

The fundamental requirement is a change in culture and thinking in our community. To facilitate getting over the initial hurdles of this culture change, tools that allow this transition are required.

kylebaron replied:
I'm not disagreeing with anything so far ... just bringing some nuance in the comparison with other fields: 

__Regarding comparison with progress in other fields__: do they have anything equivalent to regulatory agencies in their field?  I mean it's one thing to try some new tool or method in analyzing your data, writing it up as a manuscript, submitting it to the journal with the publication on the line.  But another thing to do the work and submit to regulatory agency with the approval on the line.  In the former case, you might be praised and rewarded for novelty and innovation and taking a risk.  In the latter case ... I don't know.  You might be opening yourself up to more scrutiny.  Maybe not.  

I get a lot of questions about how FDA views `mrgsolve`.   I can say we've used it in submission work without issue before.  It's not that they asked and said "ok" or they asked and had a lot of questions.  *No one has ever asked*.  So I just don't worry about it.  I suspect enough people working in industry still worry about it too much?  

I'm just saying that maybe the culture change needs to take into account the influence of regulators in our work (regardless of *where* you work) that might not be present in other fields.

vjd replied:
Great point @kylebmetrum ! To that point, my response would be the following -
Other than a few folks like yourself and Ron etc, people have not even figured out the possibilities. As a community, if we work together to bring tools into the open and work more collaboratively, and actually produce material with substance, then we can worry about how the regulators or others would accept. 
It comes down to pushing through these hurdles to show that good work can be done collaboratively

MikeKSmith replied:
Thanks for the input @kylebmetrum and @vjd. 

The point addressing "fear, uncertainty and doubt" around using open-source software should probably be addressed in a separate blog post! ;-)  I could write a whole separate post on that one. Maybe I should slim down text pertaining to that here?

The main point of this blog is really about breaking out of the mindset that the models we develop are only of interest *within* the pharmacometrics community and so there's no need for new language standards. The open-source bit is really the cherry on top, as the community itself can contribute, refine and evolve the language.

kylebaron replied:
@MikeKSmith Agreed.  My comment was a bit off topic with respect to the original blog post. ... more interacting with @vjd comment about comparison with other fields.

MikeKSmith replied:
Happy to receive a blog post on this topic anytime you like!! ;-)

Commit:
Update DDMoRe_Language_Community.Rmd

Minor revisions following review by Chris Shilling.

Commit:
Update But_I_only_use_NONMEM.Rmd

Added Vijay's comment about the impact of open-source standards on other quantitative disciplines.

Commit:
Update DDMoRe_Language_Community.Rmd

Add acknowledgement to Chris Shilling for input.

MikeKSmith commented:
Added these two blog posts following revisions. Will schedule these for posting.
